conf.py

```diff
@@ -104,6 +105,14 @@
 
 todo_include_todos = True
 
+# sphinxext-opengraph config
+ogp_site_url = "https://devguide.python.org/"
```

Comment:
```suggestion
ogp_site_url = "https://devguide.python.org/"
ogp_site_name = "Python Developer's Guide"
```
I think this would be useful to add:
> `ogp_site_name`:
>     This is not required. Name of the site. This is displayed above the title.

In this example this would be the small "GitHub" above the title.
![image](https://user-images.githubusercontent.com/25624924/194960638-86337e59-84aa-4d82-8331-08bf8501c890.png)

Compare with this (old) preview, that doesn't mention "Devguide" anywhere:
![image](https://user-images.githubusercontent.com/25624924/194961064-9caa7b57-2302-46a1-9f26-a37caae304df.png)